dat/BIOS.dat

```diff
@@ -93,6 +99,7 @@ game (
 	name "Nintendo - Nintendo Entertainment System"
 	description "Nintendo - Nintendo Entertainment System"
 	rom ( name NstDatabase.xml size 1009534 crc ebb2196c md5 7bfe8c0540ed4bd6a0f1e2a0f0118ced sha1 26322f182540211e9b5e3647675b7c593706ae2b )
+	rom ( name disksys.rom size 8192 crc 5e607dcf md5 ca30b50f880eb660a320674ed365ef7a sha1 57fe1bdee955bb48d357e463ccbf129496930b62 )
```

Comment:
I understand disksys.rom comes from Famicon system.

There are two NES cores that uses Famicon right now.

http://wiki.libretro.com/index.php?title=Nintendo_NES_Core_Compatibility

>"Note: Nestopia and FCEUmm are the only cores with fully functional Famicom Disk System support."

The wikipedia page also have both of them under NES too.

https://en.wikipedia.org/wiki/Nintendo_Entertainment_System

>At June 1985's Consumer Electronics Show (CES), Nintendo unveiled the American version of its Famicom, with a new case redesigned by Lance Barr and featuring a "zero insertion force" cartridge slot.[22] This is the system which would eventually be officially deployed as the Nintendo Entertainment System, or the colloquial "NES".

They're pretty much same... I see that we do not have a Famicon core so if you're going to play some Famicon games, you will have to use one of two NES cores.

According to the display_name in all NES/Famicom cores (bnes, emux_nes, fceumm, nestopia, and quicknes), they contains `NES / Famicom (coreName)`.

According to database in `fceumm_libretro.info`, `nestopia_libretro.info`, and `quicknes_libretro.info`, maybe the name should be... `Nintendo - Nintendo Entertainment System|Nintendo - Famicom Disk System` although I'm not sure how clrmamepro and other rom managers would react to that. 😄 

How should we solve this?